simulator/src/sequential/SRflipFlop.js

```
@@ -51,6 +51,9 @@ export default class SRflipFlop extends CircuitElement {
      * always resolvable
      */
     isResolvable() {
+        if(this.reset.value == 1 && this.preset.value == 1 || this.S.value == 1 && this.R.value == 1){
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)